soh/soh/Enhancements/randomizer/hook_handlers.cpp

```diff
@@ -1667,9 +1667,6 @@ void RandomizerOnVanillaBehaviorHandler(GIVanillaBehavior id, bool* should, va_l
         case VB_GIVE_ITEM_STRENGTH_1:
         case VB_GIVE_ITEM_ZELDAS_LETTER:
         case VB_GIVE_ITEM_OCARINA_OF_TIME:
-        case VB_GIVE_ITEM_KOKIRI_EMERALD:
```

Comment:
couldn't find a `GameInteractor_Should` that uses this https://github.com/search?q=repo%3AHarbourMasters%2FShipwright+VB_GIVE_ITEM_KOKIRI_EMERALD&type=code

Reply:
looks like this was added in https://github.com/HarbourMasters/Shipwright/commit/596ea5ebbb3e5a5dfab72fb7bee6d4802d8b13d4 but i couldn't find it being used in there either